### ISSUE TITLE
Object API model validation is broken

### ISSUE THREAD

**Issue by xxshady** [altmp/altv-issues#1659]
### Description of the problem

/

### Reproduction steps

```js
new alt.Object("awdawdwwd", alt.Player.local.pos, alt.Vector3.zero) // crash
native.createObjectNoOffset("awdawdwwd", 0, 0, 0, false, false, false) // no crash
```

### Expected behaviour

The model should be validated.

### Operating system

Windows 11

### Version

13.0-dev44

**Comment by litolax** [comment]
Confirmed on 13.0-dev44

**Comment by C0kkie** [comment]
Fixed in  dev/13.0-dev45
